packages/pn-personafisica-webapp/src/component/Contacts/SpecialContactElem.tsx

```diff
@@ -116,6 +116,12 @@ const SpecialContactElem = memo(({ address, senders, recipientId }: Props) => {
     });
   }, [address]);
 
+  const resetModifyValue = () => {
```

Comment:
Maybe this is useless

Reply:
I should modify the function updateContact to accept a second optional parameter (the id of the field) and, in case of 'cancelled', I should reset the field.

Something like that
`  const updateContact = async (status: 'validated' | 'cancelled', id: string) => {
    if (status === 'cancelled') {
      await formik.setFieldValue(id, initialValues[id]);
    }
    trackEventByType(TrackEventType.CONTACT_SPECIAL_CONTACTS, { action: EventActions.ADD });
  };`